xml/System.Diagnostics/DebuggerDisableUserUnhandledExceptionsAttribute.xml

```diff
@@ -20,8 +20,39 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>
+      <para>If a .NET Debugger is attached which supports the Debugger.BreakForUserUnhandledException(Exception) API, this attribute will prevent the debugger from breaking on user-unhandled exceptions when the exception is caught by a method with this attribute, unless BreakForUserUnhandledException is called.</para>
+    </summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[
+
+        ## Remarks
+        Visual Studio has added support for catching asynchronous user-unhandled exceptions and is enabled by default. This feature has existed for a long time for synchronous methods, but not for async / await methods. The BreakForUserUnhandledException disables the feature for specific methods. This is useful for exceptions that propagate through user code but are expected to be handled by framework code. This attribute is designed to be used along with <xref:System.Diagnostics.Debugger.BreakForUserUnhandledException(System.Exception)>.
+
+        ## Example
+
+        ```csharp
+        [MethodImpl(MethodImplOptions.NoInlining)]
+        [DebuggerDisableUserUnhandledExceptions]
+        static async Task InvokeUserCode(Func<Task> userCode)
+        {
+          try
+          {
+              await userCode();
+          }
+          catch (Exception ex)
+          {
+              if (TryHandleWithFilter(ex))
+              {
+                  return; // example case where we don't want to break for user-unhandled exceptions
+              }
+
+              Debugger.BreakForException(e); // debugger will stop here and show the exception if attached.
```

Comment:
Should this be BreakForUserUnhandledException?

Reply:
Correct - I've pushed a fix